recipe/patches/0001-Fix-TZPATH-on-windows.patchFrom a9d9b98f645ed67923d299c5e03537abe50951df Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Wed, 7 Oct 2020 10:08:30 -0500
Subject: [PATCH] Fix TZPATH on windows

---
 lib-python/3/sysconfig.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib-python/3/sysconfig.py b/lib-python/3/sysconfig.py
index a2eb52f..7dbb90c 100644
--- a/lib-python/3/sysconfig.py
+++ b/lib-python/3/sysconfig.py
@@ -585,7 +585,7 @@ def get_config_vars(*args):
 
         if os.name == 'nt':
             _init_non_posix(_CONFIG_VARS)
-            _CONFIG_VARS['TZPATH'] = ''
+            _CONFIG_VARS['TZPATH'] = os.path.join(_PREFIX, "share", "zoneinfo")
         if os.name == 'posix':
             _init_posix(_CONFIG_VARS)
         # For backward compatibility, see issue19555
-- 
2.42.0

